report/report.go

```diff
@@ -188,7 +188,9 @@ type Report struct {
 	// Job represent all Kubernetes Job on hosts running probes.
 	Job Topology
 
-	DNS DNSRecords `json:"nodes,omitempty" deepequal:"nil==empty"`
+	DNS DNSRecords `json:"DNS,omitempty" deepequal:"nil==empty"`
+	// For release 1.11.6, probes accidentally sent DNS records labeled "nodes".
```

Comment:
Could you please link https://github.com/weaveworks/scope/commit/951629af292ad2964a73766840d8d2e3ac983e99 in the comment? Just so that it's easier to track how much time has passed.

Related to that, how long do you think we should keep `BugDNS` field around? Hopefully not forever :O

Reply:
Not forever, but If 1.11.6 is the last version that supports pre-1.9 Kubernetes we probably need to keep compatibility for a year or two.